src/hammer-vlsi/hammer_vlsi/constraints.py

```diff
@@ -248,6 +242,76 @@ def to_dict(self) -> dict:
     ('assignments', List[BumpAssignment])
 ])
 
+class BumpsPinNamingScheme(Enum):
+    A0 = 0
+    A1 = 1
+    A00 = 2
+    A01 = 3
+
+    @classmethod
+    def __mapping(cls) -> Dict[str, "ObstructionType"]:
+        return {
+            "A0": BumpsPinNamingScheme.A0,
+            "A1": BumpsPinNamingScheme.A1,
+            "A00": BumpsPinNamingScheme.A00,
+            "A01": BumpsPinNamingScheme.A01
+        }
+
+
+    def name_bump(self, definition: BumpsDefinition, assignment: BumpAssignment) -> str:
+        # Skip I, O, Q, S, X, and Z
+        letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'J', 'K', 'L', 'M', 'N', 'P', 'R', 'T', 'U', 'V', 'W', 'Y']
+        # Sanity check
+        assert len(letters) == 20
```

Comment:
Is it too clunky to instead assign `len(letters)` to a variable and use that throughout?

Reply:
oh, yeah.

Reply:
fixed